### PR TITLE
chore: Labs

### PR DESCRIPTION
**Pull request by gbarkhatov**
Chain -> Labs

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -7,10 +7,10 @@ License text copyright (c) 2017 MariaDB Corporation Ab, All Rights Reserved.
 
 Parameters
 
-Licensor: Babylonchain, Inc.
+Licensor: Babylon Labs, Ltd.
 
 Licensed Work: btc-staking-ts
-The Licensed Work is (c) 2023 Babylonchain, Inc.
+The Licensed Work is (c) 2024 Babylon Labs, Ltd.
```
Comment (vitsalis) [inline LICENSE:13]: Need to verify whether this can be updated to 2024.

Comment (gbarkhatov) [reply LICENSE:13]: Also, lines should be "-----------------------------------------------------------------------------" @vitsalis ?

Comment (gbarkhatov) [reply LICENSE:13]: @vitsalis should I exclude license changes from the current PR, merge it, then create a separate PR once we get the info about the License?

Same for the `countdown`?

File: package.json [hunk 2]
```diff
@@ -41,7 +41,7 @@
       "eslint --fix"
     ]
   },
-  "author": "Babylonchain Inc.",
+  "author": "Babylon Labs, Ltd.",
```
Comment (vitsalis) [inline package.json:44]:
```suggestion
  "author": "Babylon Labs Ltd.",
```

**Review by vitsalis — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (gbarkhatov) [reply package.json]: Done

File: LICENSE (hunk 1 above)
Comment (gbarkhatov) [reply LICENSE]: Done

**Pull request merged**